proto/tokenfactory/query.proto

```diff
@@ -14,23 +14,23 @@ service Query {
   // Params defines a gRPC query method that returns the tokenfactory module's
   // parameters.
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
-    option (google.api.http).get = "/osmosis/tokenfactory/v1beta1/params";
+    option (google.api.http).get = "/sei-protocol/seichain/tokenfactory/params";
```

Comment:
I noticed most modules have `seichain` in the path but oracle has `sei-chain`, we should probably fix that for oracle, right? (not actionable for you @Kbhat1, asking @LCyson)